src/test/java/core/basesyntax/strategy/PurchaseOperationTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Storage;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationTest {
+    private static final FruitRecordDto BANANA =
```

Comment:
```suggestion
    private static final FruitRecordDto banana =
```
class is not deeply immutable so we use regular naming here